src/components/capture/useDiscoverCapture.ts

```diff
@@ -69,7 +68,7 @@ import {
 } from 'utils/workflow-utils';
 
 const trackEvent = (payload: any) => {
-    LogRocket.track(CustomEvents.CAPTURE_DISCOVER, {
+    logRocketEvent(CustomEvents.CAPTURE_DISCOVER, {
```

Comment:
This will be all over this PR. Moving this call to a single function so we can make it extra safe.